README.rst

```diff
@@ -0,0 +1,77 @@
+pyincore
+========
+
+**pyIncore-incubator** is a component of IN-CORE. It is a python package
+consisting of new analyses that build on pyIncore, but are not yet part of the
+core analyses.
+
+Installation with conda
+-----------------------
+
+Installing **pyincore** with Conda is officially supported by IN-CORE development team. 
```

Comment:
Same here, the instruction seems to be about installing pyincore-incubator but here it says "pyincore". 
Also is the dev team gonna support "pyincore-incubator"?

Reply:
That's a good question. I talked with Jong a little and I think the idea would be that we will support the conda installation and maintaining that it works and publishes packages on some release schedule, but contributors would be responsible for making sure their code still works with new releases.